.travis.yml

```diff
@@ -47,15 +53,7 @@ jobs:
     script:
       - docker run -dit --privileged --name systemd --mount type=bind,source="$(pwd)",target=/integrated-manager-for-lustre  -v /sys/fs/cgroup:/sys/fs/cgroup:ro centos/systemd
       - docker exec -i systemd bash -c "./integrated-manager-for-lustre/tests/framework/services/runner.sh"
-  - stage: cd
-    if: branch !~ ^v\d+\.\d+\.\d+-.+$ AND type NOT IN (push, pull_request)
-    env: 'Continuous Deployment'
-    script: skip
-    before_deploy:
-    - include/travis/copr-deploy.sh prepare
-    deploy:
-      skip_cleanup: true
-      provider: script
-      script: ./travis_wait "./include/travis/run_in_centos7_docker.sh include/travis/copr-deploy.sh build_srpm"
-      on:
-        all_branches: true
\ No newline at end of file
+stages:
```

Comment:
This is very interesting. How does this work?

Reply:
I noticed the conditions weren't being applied to the stage and came across this document: https://docs.travis-ci.com/user/build-stages/#Specifying-Stage-Order-and-Conditions